### PR TITLE
[Project Submission] Flink- Filecoin and Chainlink Data Provider

### PR DESCRIPTION
**Pull request by flyworker**

### CODE REVIEW

File: templates/flink.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Project Name <!-- Add your project name here with format "Project Name"-->
```
Comment (longfeiWan9) [inline templates/flink.md:1]: Please move this file to the right folder, not in templates.

File: templates/flink.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    Chainlink external adapter
+    Data DAO notary
+
+Several blockchain scanner aggeragate the data to a unified data provider, the chainlink esternal adapter with the oracle smartcontracs broadcasting the proof to target blockchain networks. Data notaries will sign the transaction based the oracle data onchain informtion.The payment is triggered once the proof that it has been stored on the Filecoin network is relayed from Chainlink oracles.
+
+## Project Status
+ beta
+
+## Previews
+![image](https://user-images.githubusercontent.com/8363795/155874454-d1b0b273-eb2f-41ec-b416-da11f466d2ba.png)
```
[the remark this answers is not in the document]
Comment (flyworker) [reply templates/flink.md:21]: The multi-chain storage use flink as the backend service to get the deal information from smart contract after storage deal is onchain. Flink it self is a chainlink oracle external service return the lotus deal info, which do not have an UI

File: templates/flink.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+### Team Size  
+3
+### Team members  
+
+    Yiming Liu Smart Contract Developer GitHub: @liushmh
+    Rick Zhao Blockchain Developer GitHub: @rykci
+
+## How the community can engage
+GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->
```
[the remark this answers is not in the document]
Comment (flyworker) [reply templates/flink.md:56]: Yes , it is here
https://github.com/filecoin-project/community/discussions/480

File: templates/project-submission-template.md [hunk 4]
```diff
@@ -1,47 +0,0 @@
-## Project Name <!-- Add your project name here with format "Project Name"-->
```
[the remark this answers is not in the document]
Comment (flyworker) [reply templates/project-submission-template.md]: OK, i take it back

File: templates/flink.md (hunk 1 above)
Comment (flyworker) [reply templates/flink.md]: Move to the dev tools

**Review by longfeiWan9 — APPROVED** [review]

**Pull request merged**